addons/account/wizard/account_move_send.py

```diff
@@ -45,9 +54,18 @@ class AccountMoveSend(models.TransientModel):
     send_mail_readonly = fields.Boolean(compute='_compute_send_mail_extra_fields')
     mail_template_id = fields.Many2one(
         comodel_name='mail.template',
-        string="Use template",
+        string="Email template",
         domain="[('model', '=', 'account.move')]",
     )
+    pdf_template_id = fields.Many2one(
+        comodel_name='ir.actions.report',
+        string="Invoice template:",
+        domain="[('is_invoice_report', '=', True)]",
+        default=_get_default_pdf_template_id,
+    )
+    show_pdf_template_menu = fields.Boolean(
```

Comment:
put this in one line no need to split it since there is only one parameter :smile: